src/scss/components/form/_index.scss

```diff
@@ -1 +1,156 @@
+@import "config";
+
+// for Firefox
 @import "reset"; // ./_reset.scss
+
+
```

Comment:
Please remove the extra white space line number :6

Reply:
Extra space removed.